source/distributed/tria.cc

```diff
@@ -45,10 +45,10 @@ namespace
   void
   get_vertex_to_cell_mappings(
     const Triangulation<dim, spacedim> &triangulation,
-    std::vector<unsigned int>          &vertex_touch_count,
+    std::vector<unsigned int> &         vertex_touch_count,
```

Comment:
these changes mean that you did not run "make indent" and pushed your changes again.